src/app/features/tech-record/tech-record.component.html

```diff
@@ -5,9 +5,7 @@
   ></app-vehicle-technical-record>
 </ng-container>
 
-<h1 *ngIf="(spinnerService.showSpinner$ | async) === false && (vehicleTechRecord$ | async) === undefined">
-  This vehicle is not real, like the no vehicle page we need to create
-</h1>
+<h1 *ngIf="(vehicleTechRecord$ | async) === undefined">This vehicle is not real, like the no vehicle page we need to create</h1>
```

Comment:
Consider using the loose equality operator (`==`) to be on the safe side, in case `vehicleTechRecord$` returns a `null` instead of `undefined`.

Reply:
Lint rules are we have to use `===` and the data is of the tech record model or undefined.

Reply:
I see, resolving then.